wordpress/audits.json

```diff
@@ -75,7 +75,7 @@
     "id": "uses-responsive-images",
     "title": "Properly size images",
     "description": "Serve images that are appropriately-sized to save cellular data and improve load time. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/oversized-images).",
-    "secondaryDescription": "Upload images directly through the media library to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). "
+    "secondaryDescription": "Upload images directly through the media library to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage."
```

Comment:
Any chance there's a canonical link within WordPress that we can point to that describes full size/full width images?

Reply:
Sure is! We can link it to this: https://codex.wordpress.org/Inserting_Images_into_Posts_and_Pages#Image_Size

I'll update the pull request.